client/src/controllers/SkipLinkController.ts

```diff
@@ -0,0 +1,35 @@
+import { Controller } from '@hotwired/stimulus';
+
+/**
+ * Appears at the top left corner of the admin page with the tab button is clicked.
+ * Used to provide an accessible skip button for keyboard control so that users can
+ * easily navigate to the main content without having to navigate a long list of navigation links.
+ *
+ * Inspired by https://github.com/selfthinker/dokuwiki_template_writr/blob/master/js/skip-link-focus-fix.js
+ *
+ */
+
+export class SkipLinkController extends Controller<HTMLElement> {
+  skipToTarget!: HTMLElement | null;
```

Comment:
I am concerned that this not null `!` is not really the best solution here - it is kind of an escape hatch and does not really help solve the root problem here.

The problem here is that we have introduced `strictPropertyInitialization` and this is forcing us to align with TypeScript's expectations of the constructor only being the place where instance properties will be set up.

However, this will conflict with Stimulus and any other approach where this is not the case, also we have `| null` here which actually means this can be null.

So I will suggest we do the following:

```suggestion
  skipToTarget?: HTMLElement | null;
```
This says that the property may not exist (`?` at the start) and when it does it could be a HTML element or a null (from query selector).

This also means we can rely on the type safety in other code that references it in the same file.

Finally, I will turn off the tsconfig item for now - I think we need more discussion about whether `strictPropertyInitialization` is something we want on in the codebase @thibaudcolas